openshift-ci/Dockerfile.index.ci

```diff
@@ -0,0 +1,6 @@
+FROM quay.io/operator-framework/upstream-opm-builder
```

Comment:
where is this Dockerfile.index.ci image used ?

Reply:
Here: https://github.com/openshift/release/pull/10474/commits/d17a2bf097412800daaa6ddeee44f9f9e3d00868#diff-fb490d3ebe4d085e38b1a4faf94429adR46

Reply:
The issue here is the nested path. the ".ci" variants have a flatter path which save us quit a lot of complications in openshift CI.